webanno-ui-curation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/curation/page/CurationPage.java

```diff
@@ -827,13 +826,10 @@ public void onClick(AjaxRequestTarget aTarget)
                 CAS cas = curationDocumentService.readCurationCas(state.getDocument());
                 updateCurationView(curationContainer, curationViewItem, aTarget,
                         cas);
-                updatePanel(aTarget, curationContainer);
+//                updatePanel(aTarget, curationContainer);
```

Comment:
Why is it not needed anymore? Why not entirely remove it instead of commenting out?

Reply:
`updateCurationView` calls `updatePanel`. I only commented it out though in case I missed something and we find that this messes something up upon more testing